src/components/map/map.js

```diff
@@ -83,6 +83,7 @@ export default function Map({
   state = null,
   popoutAbbreviation = null,
   setPopoutAbbrevation,
+  zoomControl = true,
```

Comment:
Is this supposed to control the zoom in and out? For me, it works for zooming in on the state map but not zooming out for some reason.

Reply:
Looking back at the design, the home page should allow zooming and the state/opo pages should not. I will set this default value to false, and the index page will pass true.